lib/samples/show_legend/show_legend_sample.dart

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+import '../../utils/sample_state_support.dart';
+
+class ShowLegendSample extends StatefulWidget {
+  const ShowLegendSample({super.key});
+  @override
+  State<ShowLegendSample> createState() => _ShowLegendSampleState();
+}
+
+class _ShowLegendSampleState extends State<ShowLegendSample>
+    with SampleStateSupport {
+  // create a map view controller
+  final _mapViewController = ArcGISMapView.createController();
+  // create a map with a basemap style
+  final _map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+  // create a variable to store the selected legend
+  LegendInfo? _selectedLegend;
+  // create a variable to store the legend image
+  ArcGISImage? _arcGISImage; 
+  // create a list to store dropdown items
+  final _legendsDropDown = <DropdownMenuItem<LegendInfo>>[];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Column(
+          // add the map view and dropdown button to a column.
+          children: [
+            Expanded(
+              // add a map view to the widget tree and set a controller.
+              child: ArcGISMapView(
+                controllerProvider: () => _mapViewController,
+                onMapViewReady: onMapViewReady,
+              ),
+            ),
+            // add a dropdown button to the widget tree.
+            DropdownButtonHideUnderline(
+              child: DropdownButton(
+                value: _selectedLegend,
```

Comment:
I think you can just delete this line and delete where `_selectedLegend` is declared further up. The `value` parameter doesn't have to be specified.

Reply:
I can remove the value param from the DropdownButton widget but if I remove it from the DropdownMenuItem widgets it crashes. Is it fine if we pass value for each DropdownMenuItem but not for DropdownButton?

Reply:
Yeah I think that's fine.